picard/util/thread.py

```diff
@@ -36,10 +36,10 @@ def run(self):
 
 class Runnable(QRunnable):
 
-    def __init__(self, func, next):
+    def __init__(self, func, next_func):
         QRunnable.__init__(self)
         self.func = func
-        self.next = next
+        self.next = next_func
 
```

Comment:
I would prefer if the lhs variable is also self.next_func, just to maintain consistency